.gitignore

```diff
@@ -4,3 +4,7 @@ docker/sgx
 make.log
 cscope.files
 cscope.out
+ccf_transaction_processor/test/infra
+ccf_transaction_processor/test/*.pem
+ccf_transaction_processor/generate_signing_keys/infra
+ccf_transaction_processor/generate_signing_keys/*.pem
```

Comment:
do we need these changes? some of these directories no longer exist.

Reply:
Not needed after the refactoring. Will remove them. Thank you.